cmd/arcaflow/main.go

```diff
@@ -136,7 +134,7 @@ Options:
 
 	logger := log.New(cfg.Log).WithLabel("source", "main")
 
-	dirContext, err := loadContext(dir)
+	dirContext, err := loadfile.LoadContext(dir)
 	if err != nil {
 		logger.Errorf("Failed to load configuration file %s (%v)", configFile, err)
```

Comment:
Outside the scope of your change, really, but this error is a duplicate of line 122, and misleading if it ever happens since the failure is in loading the regular files from your context directory rather than `configFile`.